frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -321,7 +321,7 @@ const HostSummary = ({
     if (platform === "chrome") {
       return <DataSet title="Agent" value={summaryData.osquery_version} />;
     }
-    if (summaryData.orbit_version) {
+    if (summaryData.orbit_version !== DEFAULT_EMPTY_CELL_VALUE) {
```

Comment:
slightly weird that the way we're using `normalizeEmptyValues` means this and the below check are for the string `DEFAULT_EMPTY_CELL_VALUE`, which seems like a value intended for rendering, instead of `null` or `undefined`, which would be more programmatic represantions of emptiness. For future thought.

Reply:
Yeah, default empty value was literally the default for empty values for the entire host details page. Whomever built it didn't consider how sticky it would be if we wanted to render something different. Definitely future thought since the scope creep of refactoring all that is unknown.